src/Sticky.js

```diff
@@ -85,24 +85,20 @@ export default class Sticky extends Component {
       style
     });
   };
+  
+  contentRef = content => {
+    this.content = ReactDOM.findDOMNode(content);
+  };
+  placeholderRed = placeholder => {
+    this.placeholder = placeholder
+  };
 
   render() {
-    const element = React.cloneElement(
-      this.props.children({
-        isSticky: this.state.isSticky,
-        wasSticky: this.state.wasSticky,
-        distanceFromTop: this.state.distanceFromTop,
-        distanceFromBottom: this.state.distanceFromBottom,
-        calculatedHeight: this.state.calculatedHeight,
-        style: this.state.style
-      }),
-      { ref: content => { this.content = ReactDOM.findDOMNode(content); } }
-    )
-
+    const Component = this.props.children;
     return (
       <div>
-        <div ref={ placeholder => this.placeholder = placeholder } />
-        { element }
+        <div ref={this.placeholderRef} />
+        <Component {...this.state} ref={this.contentRef}/>
```

Comment:
I like this.

Just make sure there are accompanying tests for this as well -- I'm assuming you ran the tests with the typo above and they still passed, so if you could also fix that too, that'd be awesome.